definitions/variable/industry/scrap.yaml

```diff
@@ -0,0 +1,32 @@
+
+- Collected Scrap|Iron and Steel|Steel|{Demand Sector}:
+    description: Steel scrap from {Demand Sector}
```

Comment:
In the description, we can clarify that "collected scrap" is the amount that goes to recycling.